cmd/kwild/server/build.go

```diff
@@ -65,17 +65,17 @@ func buildServer(d *coreDependencies, closers *closeFuncs) *Server {
 	db.AutoCommit(true) // alt: db.BeginTx ... defer tx.Rollback, etc.
 
 	// engine
-	e := buildEngine(d, closers, db)
+	e := buildEngine(d, db)
 
 	// account store
 	initAccountRepository(d, db)
 
 	// validator updater and store
 	//vstore := buildValidatorManager(d, db) // TODO: get rid of dis
```

Comment:
Well, let's get rid of it then :)

Reply:
haha this is my bad. I left this in from a previous PR